README.md

```diff
@@ -1,102 +1,107 @@
-<!--
-This README describes the package. If you publish this package to pub.dev,
-this README's contents appear on the landing page for your package.
+<div align="center">
 
-For information about how to write a good package README, see the guide for
-[writing package pages](https://dart.dev/guides/libraries/writing-package-pages).
-
-For general information about developing packages, see the Dart guide for
-[creating packages](https://dart.dev/guides/libraries/create-library-packages)
-and the Flutter guide for
-[developing packages and plugins](https://flutter.dev/developing-packages).
--->
-
-<p align="center">
 <img width="200" src="https://github.com/Iteo/hooked_bloc/raw/main/hooked_bloc_icon.png">
-</p>
+<br /><br />
 
+[![Build](https://github.com/Iteo/hooked_bloc/workflows/Build/badge.svg)](https://github.com/Iteo/hooked_bloc/actions?query=workflow%3ATest)
+&nbsp;
 [![codecov](https://codecov.io/gh/Iteo/hooked_bloc/branch/main/graph/badge.svg)](https://codecov.io/gh/Iteo/hooked_bloc)
+&nbsp;
+[![pub package](https://img.shields.io/pub/v/hooked_bloc.svg)](https://pub.dartlang.org/packages/hooked_bloc) &nbsp;
+[![stars](https://img.shields.io/github/stars/Iteo/hooked_bloc.svg?style=flat&logo=github&colorB=deeppink&label=stars)](https://github.com/Iteo/hooked_bloc)
+&nbsp;
+[![GitHub license](https://img.shields.io/github/license/Iteo/hooked_bloc)](https://github.com/Iteo/hooked_bloc/blob/main/LICENSE) &nbsp;
+
+</div>
+
+---
```

Comment:
I think this duplicates divider with `#Hooked bloc` title

Reply:
done